buildconfig/CMake/LinuxPackageScripts.cmake

```diff
@@ -131,6 +131,19 @@ endif()
 # ######################################################################################################################
 # Launcher scripts
 # ######################################################################################################################
+set(CONDA_PREAMBLE_TEXT
+    "# Verify that conda is setup
+if [ -z \"\${CONDA_PREFIX}\" ]; then
```

Comment:
We've been using this script without an activation of the conda environment on our analysis machines. We currently have a launcher menu item pointed at it.
It also won't work in the tarred up bundle we're creating:  https://github.com/mantidproject/mantid/releases/download/nightly/mantidworkbenchnightly-6.4.20220907.1204.tar.xz as that doesn't have a full conda environment

Can we just say use the `CONDA_PREFIX` if defined else fallback to the relative path checks?

Reply:
I did not know anything about how that tarball works and assumed it acted like a proper conda package. I'll do some more investigation.